package.json

```diff
@@ -52,11 +53,11 @@
     "react-redux": "^7.2.1"
   },
   "dependencies": {
-    "axios": "^0.21.0",
+    "axios": "^0.25.0",
```

Comment:
we don't have too good test coverage in this package - maybe publish as patch release and try to install in one of our apps? just to be sure everything is ok